assets/prefabs/structures/Smith.prefab

```diff
@@ -0,0 +1,70 @@
+{
+    "StructureTemplate": {
+    },
+    "SpawnBlockRegions": {
+        "regionsToFill": [
+            { "blockType": "CoreBlocks:DoorBottom.BACK", "region": { "min": [-2, 1, 1], "size": [2, 1, 1]}},
+            { "blockType": "CoreBlocks:DoorTop.BACK", "region": { "min": [-2, 2, 1], "size": [2, 1, 1]}},
+            { "blockType": "CoreBlocks:Glass", "region": { "min": [-4, 2, 3], "size": [1, 1, 2]}},
+            { "blockType": "CoreBlocks:Glass", "region": { "min": [4, 2, 3], "size": [1, 1, 2]}},
+            { "blockType": "CoreBlocks:Glass", "region": { "min": [2, 2, 6], "size": [1, 1, 1]}},
+            { "blockType": "HjerlHede:brownCornerRoof.BACK", "region": { "min": [-2, 6, 4], "size": [1, 1, 1]}},
```

Comment:
Given my comment above, all occurrences of `HjerlHede:brownCornerRoof` can be replaced by `hjerlhede:brownroof:engine:slopecorner` in the structural template prefab.